app/editor/src/features/admin/media-types/styled/MediaTypeList.tsx

```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components';
+
+export const MediaTypeList = styled.div`
+  width: 100%;
```

Comment:
Don't think we should use `100%` ever.  I think `auto` is safer.  But I'm not an expert in this area.  Just finding issues when we do it this way.